src/include/common/hip/rpp_hip_math.hpp

```diff
@@ -273,11 +273,11 @@ __device__ __forceinline__ float rpp_hip_math_inverse_sqrt1(float x)
 
 __device__ __forceinline__ float4 rpp_hip_math_inverse_sqrt4(float4 x_f4)
 {
-    float4 xHalf_f4 = (float4)0.5f * x_f4;
+    float4 xHalf_f4 = MAKE_FLOAT4(0.5f) * x_f4;
     int4 i_i4 = *(int4 *)&x_f4;                                     // float bits in int
-    i_i4 = (int4) NEWTON_METHOD_INITIAL_GUESS - (i_i4 >> (int4)1);  // initial guess for Newton's method
+    i_i4 = MAKE_INT4(NEWTON_METHOD_INITIAL_GUESS) - (i_i4 >> MAKE_INT4(1));  // initial guess for Newton's method
```

Comment:
i_i4 is confusing. If x_f4 is float4 version of x, x_i4 should be int4 version of x right?

There are only 3 funcitons in this file that take x as an argument. I would change all of x to "val".
Then use val_f4, val_i4 etc